### ISSUE TITLE
Does this work with display: flex ? 

### ISSUE THREAD

**Issue by jcklpe** [davatron5000/FitText.js#165]
I'm fixing to check and see if it does, but wanted to put up this question for the sake of documentation of future users.

**Comment by CHEWX** [comment]
@thedonquixotic Yes, I am having no adverse affects.

**Comment by derSteffen** [comment]
Yes, there are problems with display:flex. If the fittext-element in any div (and any parent-div has display:flex) the compressor doesn't work exactly. The scale-effect (smaller) ist very fast and then there ist no scale-effect to make it bigger.

My soluion was to give the fittext-element an extra parent div. And this div I only give them a width:100% and a max-width:auto

But this is not the solution for the future.
